gpflow/utilities/utilities.py

```diff
@@ -228,12 +228,21 @@ def reset_cache_bijectors(input_module: tf.Module) -> tf.Module:
     target_types = (tfp.bijectors.Bijector, )
     accumulator = ('', None)
 
+    def clear_cache(b):
+        if isinstance(b, tfp.bijectors.Bijector):
+            # `_from_x` and `_from_y` are cache dictionaries for forward and inverse transformations
+            # in bijector class.
+            b._from_x.clear()
+            b._from_y.clear()
+
     def clear_bijector(bijector, _, state):
-        bijector._from_x.clear()
-        bijector._from_y.clear()
+        clear_cache(bijector)
+        if isinstance(bijector, tfp.bijectors.Chain):
+            for m in bijector.submodules:
+                clear_cache(m)
         return state
 
-    _ = traverse_module(input_module, accumulator, clear_bijector, target_types)
+    traverse_module(input_module, accumulator, clear_bijector, target_types)
```

Comment:
Is it not clearer to show that you are intentionally not making use of `traverse_module`'s return value?